docker-compose-prod.yaml

```diff
@@ -29,8 +28,6 @@ services:
         IS_REVERSY_PROXY_USED: "true"
```

Comment:
are we using reverse-proxy still? I'm pretty sure we shouldn't in current server configuration. Especially in prod env.